config/testgrids/kubernetes/kops/config.yaml

```diff
@@ -43,12 +43,7 @@ dashboards:
 - name: kops-distro-u1804
 - name: kops-distro-u2004
 - name: kops-k8s-1.17
-  days_of_results: 90
```

Comment:
This change is due to stricter YAML parsing, right? It's fine to delete these only If these entries don't currently do anything.

Looking at a [dashboard](https://testgrid.k8s.io/kops-k8s-1.17#kops-grid-kopeio-u2004-k17), it might be the case that it's only looking 30 days back, but it's tough to tell.

Reply:
We deleted this because it was currently not doing anything, and it will no longer work with the new strict parsing.